project_closing/models/project.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from odoo import models, api
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    @api.multi
+    def toggle_active(self):
+        for record in self:
+            record.analytic_account_id.active = not record.active
```

Comment:
@dreispt I disagree: toggle_active is defined in the base model, therefore it's available for all the inherited models (which basically means all the models), but it's still possible (and in cases such as this covered by this module, needed) to extend its behavior for single models.
In our case we need to add further logic, which applies only when the method toggle_active is invoked by the project.project model. This is correctly addressed by the committed code, as far as I can see. I just think it should make use of the method which is deputed to apply the archive/unarchive logic (toggle_active) also for the account.analytic.account model.